packages/host/app/components/operator-mode/card-url-bar.gts

```diff
@@ -117,14 +119,22 @@ export default class CardURLBar extends Component<Signature> {
   @service declare operatorModeStateService: OperatorModeStateService;
 
   urlBar: URLBarResource = urlBarResource(this, () => ({
-    getValue: () => this.codePath,
+    getValue: () => this.initValue,
+
     setValue: (url: string) => {
       this.operatorModeStateService.updateCodePath(new URL(url));
     },
     setValueError: this.args.loadFileError,
     resetValueError: this.args.resetLoadFileError,
   }));
 
+  get initValue() {
+    if (isReady(this.args.openFile)) {
+      return this.args.openFile.url;
+    }
+    return this.codePath;
```

Comment:
these can be different right? if that's the case we need to solve the root of state instead of tyring to spackle over the issue